src/main/java/org/reaktivity/nukleus/kafka/internal/stream/NetworkConnectionPool.java

```diff
@@ -2456,7 +2463,7 @@ public int hashCode()
         @Override
         public String toString()
         {
-            return format("id=%d, offset=%d, refs=%d", id, offset, refs);
+            return format("{id=%d, offset=%d, refs=%d}", id, offset, refs);
```

Comment:
May be you can use '(' and ')' as Map.toString() uses '{}'

Reply:
Done